paper.tex

```diff
@@ -137,7 +137,7 @@ \subsection{Bytes}
 
 The attribute \adeff{slice} represents a sub-sequence inside the current one.
 
-The attribute \adeff{concat} is a new sequence, which is a concatenation of two byte sequences: the current and the provided one.
+The attribute \adeff{chained} is a new sequence, which is a concatenation of two byte sequences: the current and the provided one.
```

Comment:
@EugeneDar this one we don't change, I believe. Or do we? @Graur WDYT?

Reply:
@yegor256 You ask to change it here https://github.com/objectionary/on-the-origin-of-objects/pull/19#discussion_r966788425
I'm ok with `chained` naming